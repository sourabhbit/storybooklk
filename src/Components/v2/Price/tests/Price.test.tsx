import { render, screen } from '@testing-library/react'
import Pricing from '../index'

const props = {
	amount: '999',
	special: false,
	id: '',
}

describe('<NumberedStar />', () => {
	let wrapper: any = null

	beforeEach(() => {
		const { asFragment } = render(<Pricing {...props} />)
		wrapper = asFragment
	})

	afterAll(() => {
		wrapper = null
	})

	it('Should take a snapshot', () => {
		expect(wrapper()).toMatchSnapshot()
	})
	it('Should render Price Component', () => {
		const container = screen.getByTestId('pricing-test')
		expect(container).toBeInTheDocument()
	})
})
