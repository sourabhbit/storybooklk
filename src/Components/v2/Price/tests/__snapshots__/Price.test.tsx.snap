// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NumberedStar /> Should take a snapshot 1`] = `
<DocumentFragment>
  <span
    class="sc-bdvvtL cJIrfk"
    data-testid="pricing-test"
  >
    999
    <span
      class="sc-gsDKAQ logYvF"
      color="var(--warm-grey-50)"
    />
  </span>
</DocumentFragment>
`;
